### DISCUSSION TITLE
Grove - 4-Channel SPDT Relay | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#70] [Q&A]
# Grove - 4-Channel SPDT Relay | Seeed Studio Wiki

The Single Pole Double Throw SPDT relay is quite useful in certain applications because it has one common terminal and 2 contacts which are great for selecting between two options. The Grove - 4-Channel SPDT Relay has four single pole - double throw (SPDT) switches. It only requires low-voltage and low current signals to control those switches. Specifically, you can use 5V DC to control max.250V AC or 110V DC. The I2C address is changeable so that you can use multiple relay modules in the same project. The Grove - 4-Channel SPDT Relay has four single pole - double throw (SPDT) switches. It only requires low-voltage and low current signals to control those switches. Specifically, you can use 5V DC to control max.250V AC or 110V DC.

https://wiki.seeedstudio.com/Grove-4-Channel_SPDT_Relay/

**Comment by Marelli7** [comment]
looks like "CHANNLE1_BIT" is a typo. Do I need to correct CHANNLE# into CHANNEL# in multi_channel_relay.h and all *.ino files?

**Reply by MatthewJeffson** [reply]
Hi, indeed there is a typo, really appreciated!

**Reply by MatthewJeffson** [reply]
If you corrected it and test it fine in the related .h and .ino files, please help us to change it and Pull Request to us !

**Comment by lelez1000** [comment]
Hi,

in case I have more than one board connected to the arduino, how could I manage both of them?

**Reply by xzxcessarr** [reply]
Hi, thank you for feedback! It is possible that if you wanna use one more board by I2C, because they have the correspond I2C address, when you use the function scanI2CDevice() you will find them
